Error: Use `aria-describedby` to reference a group description.
Error: For usability, it is recommended to put the group description at the top of the component, just after the label.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar).
Error: `aria-invalid="true"` is used to accessibly indicate fields that have errors.
Error: Color alone should not be used to convey errors.
Error: The error text should not be hidden when the component has an error.
diff --git a/output/Checkbox Group with Validation Error/2/0b0ece1f-e7b8-4364-b530-0ddd0a813717.html b/output/Checkbox Group with Validation Error/2/0b0ece1f-e7b8-4364-b530-0ddd0a813717.html
index 806a1a0..82939f8 100644
--- a/output/Checkbox Group with Validation Error/2/0b0ece1f-e7b8-4364-b530-0ddd0a813717.html	
+++ b/output/Checkbox Group with Validation Error/2/0b0ece1f-e7b8-4364-b530-0ddd0a813717.html	
@@ -1,22 +1,16 @@
-<fieldset>
+<fieldset aria-describedby="fruits-error">
   <legend>What fruits do you like?</legend>
+  <div class="error-message" id="fruits-error">You must choose one or more fruits</div>
   <div>
-    <input type="checkbox" id="bananas" name="fruits" value="bananas">
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas" aria-invalid="true">
     <label for="bananas">Bananas</label>
   </div>
   <div>
-    <input type="checkbox" id="oranges" name="fruits" value="oranges">
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges" aria-invalid="true">
     <label for="oranges">Oranges</label>
   </div>
   <div>
-    <input type="checkbox" id="apples" name="fruits" value="apples">
+    <input type="checkbox" id="apples" name="fruits[]" value="apples" aria-invalid="true">
     <label for="apples">Apples</label>
   </div>
-  <div class="error-message">You must choose one or more fruits</div>
-</fieldset>
-<style>
-  .error-message {
-    color: red;
-    display: none;
-  }
-</style>
\ No newline at end of file
+</fieldset>
\ No newline at end of file
